src/main/features/response/form/models/statement-of-means/debtRow.ts

```diff
@@ -0,0 +1,73 @@
+import { IsDefined, Min, ValidateIf } from 'class-validator'
+
+import { ValidationErrors as GlobalValidationErrors } from 'forms/validation/validationErrors'
+import { MultiRowFormItem } from 'forms/models/multiRowFormItem'
+import { Fractions } from 'forms/validation/validators/fractions'
+import { toNumberOrUndefined } from 'common/utils/numericUtils'
+import { IsNotBlank } from 'forms/validation/validators/isBlank'
+import { MaxLength } from 'forms/validation/validators/maxLengthValidator'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+
+export class ValidationErrors {
+  static readonly DEBT_REQUIRED: string = 'Enter a value'
```

Comment:
this doesn't look very specific to debts, maybe move it?

Reply:
I have talked to Tarik. All msg here will be updated